components/definition/configuration/NetPyNESimConfig.js

```diff
@@ -169,9 +273,10 @@ export default class NetPyNESimConfig extends React.Component {
           <div>
             <BottomNavigation selectedIndex={this.state.selectedIndex}>
               <BottomNavigationItem key={'General'} label={'General'} icon={<FontIcon className={"fa fa-bars"} />} onClick={() => this.select(0, 'General')} />
-              <BottomNavigationItem key={'SaveConfiguration'} label={'Save Configuration'} icon={<FontIcon className={"fa fa-floppy-o"} />} onClick={() => this.select(1, 'SaveConfiguration')} />
-              <BottomNavigationItem key={'ErrorChecking'} label={'Error Checking'} icon={<FontIcon className={"fa fa-exclamation"} />} onClick={() => this.select(2, 'ErrorChecking')} />
-              <BottomNavigationItem key={'Others'} label={'Others'} icon={<FontIcon className={"fa fa-list"} />} onClick={() => this.select(3, 'Others')} />
+              <BottomNavigationItem key={'Record'} label={'Record'} icon={RecordIcon} onClick={() => this.select(1, 'Record')} />
```

Comment:
I wouldn't use a new kind of icon... Having a look at font awesome I think you can use `<FontIcon className={"fa fa-circle"} />` It is slightly different but it should do the trick. Don't forget to delete the import and other lines that do not make sense any more